enhancements/console/quick-starts.md

```diff
@@ -151,6 +154,23 @@ resource actions. In order for the user to see the Quick Start and
 take its tour, his access review needs to pass all the listed
 resources actions.
 
+#### Quick Start Internationalization
+
+OpenShift can be toggled between multiple languages. The
+Quick Start CRD `spec` contains a `language` field to specify what
```

Comment:
Would you hide any QuickStarts that are in English when another language is selected in the UI? It would mean any QuickStarts that haven't been translated yet would be hidden entirely. I think it's probably better to show the English version if it's all we have. But if you always show English QuickStarts, then you will have duplicates potentially when those are also translated. The UI has no way to know that they're the same QuickStarts in different languages.

We might either need all of the translations in one QuickStart resource or a property that says this QuickStart translates another English QuickStart so we can hide the English one.

Reply:
Yeah, I was thinking we'd hide them, so the duplicate issue wouldn't come up. But if you want to show English ones by default, we need an approach similar to what you mentioned.